WordPress/Classes/Utility/App Configuration/AppColor.swift

```diff
@@ -86,10 +87,12 @@ struct AppColor {
 
     #if IS_JETPACK
     static let brand = CSColor.JetpackGreen.base
+    static let _brand = Color(CSColor.JetpackGreen.base)
     #endif
 
     #if IS_WORDPRESS
     static let brand = CSColor.WordPressBlue.base
+    static let _brand = Color(CSColor.WordPressBlue.base)
```

Comment:
Add `AppColor` and `UIAppColor`?

Reply:
Sure – done in c7d3a4e82ba75aba0ddee2f1d7f4d6a7f242ac54